{{cookiecutter.github_repository}}/provisioner/roles/celery/tasks/main.yml

```diff
@@ -34,4 +34,8 @@
 - name: ensure celery worker is running
   systemd: state=restarted daemon_reload=yes enabled=yes name=celery-{{ project_namespace }}
   tags: ['celery']
+
+- name: ensure celerybeat worker is running
+  systemd: state=started daemon_reload=yes enabled=yes name=celerybeat-{{ project_namespace }}
+  tags: ['celery']
```

Comment:
can you please add a `deploy` tag to it as well, the celery beat need to start only after django configurations are added. If you don't add the deploy tag it will try to run this during configure step and break.

Reply:
The same applies to the task above as well, while you are there, you can add the fix for celery worker as well.